tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -167,6 +171,38 @@
 # It is quite clear that the MRI surfaces (head, brain) are not well aligned
 # to the head digitization points (dots).
 #
+# Compute the transform
+# ---------------------
+# Let's step through the process that the transform is accomplishing.
+
+# Adjust units by * 1e3 m -> mm.
+head_space = np.array([dig['r'] for dig in
+                       raw.info['dig']], dtype=float) * 1e3
+
+# Transform digitization points into MRI space.
+mri_space = mne.transforms.apply_trans(trans, head_space, move=True)
+
+# Get landmarks in voxel space, using the T1 data.
+vox2ras_tkr = t1w.header.get_vox2ras_tkr()
+ras2vox_tkr = linalg.inv(vox2ras_tkr)
+mri_voxel_space = mne.transforms.apply_trans(ras2vox_tkr, mri_space)  # in vox
+
+fig, (ax0, ax1, ax2) = plt.subplots(1, 3)
+fig.set_size_inches(12, 3.5)
+aspect = head_space.shape[1] / head_space.shape[0]
+ax0.imshow(head_space, aspect=aspect, vmin=-100, vmax=100)
+ax0.set_ylabel('Points')
+ax0.set_title('Head')
+ax1.imshow(mri_space, aspect=aspect, vmin=-100, vmax=100)
+ax1.set_xlabel('Dimensions (x, y, z)')
+ax1.set_title('MRI RAS')
+plot = ax2.imshow(mri_voxel_space, aspect=aspect, vmin=-100, vmax=100)
+ax2.set_title('MRI Voxels')
+ax3 = fig.add_axes([0.9, 0.1, 0.03, 0.8])
+fig.colorbar(plot, cax=ax3)
+fig.show()
```

Comment:
what is it you want to show here?

for making diagrams on sphinx I would look at https://sphinxcontrib-mermaid-demo.readthedocs.io/en/latest/

Reply:
Ohhhh I didn't put it together what Mainak wanted, I can do that

Reply:
actually I've had trouble with `sphinxcontrib-mermaid`, it's not vigorously maintained (took several months for my PR to get merged, when all I was doing was bumping it to use the current stable version of mermaid).  For the ICA tutorial I used graphviz ([see here](https://github.com/mne-tools/mne-python/blob/master/doc/_static/diagrams/ica.dot))